marketplace/tests.py

```diff
@@ -93,10 +93,12 @@ def test_upload_example_data(self):
             'description': model.description,
             'category': None
         }
-        upload = open('marketplace/test_data/M1 [guinier]_out.txt', 'rb')
-        file_dict = { 'example_data': SimpleUploadedFile(upload.name, upload.read()) }
+        upload = open('marketplace/test_data/M1 [guinier]_out.txt', 'r')
+        file_dict = { 'example_data': SimpleUploadedFile(upload.name, upload.read().encode()) }
         form = SasviewModelForm(post_dict, file_dict)
 
+        print(upload.read())
```

Comment:
@krzywon I guess the print statement was added for some intermediate debugging. Should it be removed?

Reply:
Whoops. One slipped through. I'll be sure to comment out or remove this.